filters/StatsFilter.hpp

```diff
@@ -89,7 +89,7 @@ typedef std::vector<double> DataVector;
     {
         if (M2 == 0 || m_cnt <= 2 || !m_advanced)
             return 0.0;
-        double c(m_cnt);
```

Comment:
We shouldn't have to do this.  What terrible warning is MS giving?

Reply:
an implicit conversion warning.

Reply:
Wonder if we should suppress.  That seems anal.